keyboards/mechlovin/zed65/rev1/info.json

```
@@ -21,7 +21,6 @@
         "on_state": 0
     },
```

Ideally this would be on line 18, but i cannot make the suggestion there
```suggestion
    },
    "diode_direction": "COL2ROW"
```